tools/lib/h5diff_util.c

```diff
@@ -333,7 +333,7 @@ print_found(hsize_t nfound)
     if (g_Parallel)
         parallel_print("%" PRIuHSIZE " differences found\n", nfound);
     else
-        fprintf(stdout, "%" PRIuHSIZE " differences found\n", nfound);
+        fprintf(rawoutstream, "%" PRIuHSIZE " differences found\n", nfound);
```

Comment:
May also want to update `parallel_print` to use `vfprintf` to `rawoutstream` rather than `vprintf` in the serial case at least. Parallel is a little more complicated due to the multiple buffers, but `print_manager_output()` in ph5diff_main.c is generally where most of the real output comes from. It uses regular `printf` and `putchar`, which could probably be updated to `fprintf` and `fputc` as well.

Reply:
A little worried about that one.

Reply:
Yes, ph5diff is a little unique and that should probably be covered related to #5460.

Reply:
parallel_print goes to overflow_file then what prints that?

Reply:
It goes to `outBuff` first and then over a certain amount of output will go to `overflow_file`. Both are printed in `print_manager_output()` but could potentially get printed elsewhere as well.